lib/rules/media-feature-name-value-no-unknown/index.js

```diff
@@ -239,6 +240,8 @@ const rule = (primary) => {
 			};
 
 			parseMediaQuery(atRule).forEach((mediaQuery) => {
+				if (isMediaQueryInvalid(mediaQuery)) return;
```

Comment:
This rule should never warn for invalid media queries.
It should only check feature name and value pairs within grammatically valid media queries.

Otherwise it would overlap with the new rule.